### PR TITLE
Mjeter/added routes

### PR DESCRIPTION
**Pull request by mikejeter**
## Description ##

Added two routes, groomer_services and services modified migrations file to add a new column price to the services table. Modified services seed file to ad prices.
## Type ##

- [ ] This is a bugfix.
- [ ] This is a new feature.
- [x] This is part of a feature.

## Organization ##

- [x] Have you linked this pull request to a Trello card?
- [ ] Have you named the branch in the format of feature/describe_feature_or_change OR bug/describe_bug_this_is_fixing 

## Review ##

- [ ] Is this a work in progress?
- [x] Is this ready to be reviewed?
- [x] Have you added two reviewers to this pull request?

### CODE REVIEW

**Comment by mikejeter** [comment]
[create endpoint for groomer to edit/update profile information in DB](https://trello.com/c/DU59p9JM/25-create-endpoint-for-groomer-to-edit-update-profile-information-in-db)

**Comment by mikejeter** [comment]
[create endpoint to add groomer's profile information to DB](https://trello.com/c/1fwVhQHb/24-create-endpoint-to-add-groomers-profile-information-to-db)

File: data/migrations/20201014204709_create_bare_bones_tables.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ exports.up = async (knex) => {
   await knex.schema.createTable('services', (table) => {
     table.increments('id');
     table.string('service_name').notNullable();
+    table.bigint('services_price');
```
Comment (jessicagertig) [inline data/migrations/20201014204709_create_bare_bones_tables.js]: As discussed remove this column so we can format it as we all decided on.

**Comment by jessicagertig** [comment]
Closed to resolve empty file issue.

**Pull request closed without merging**